### PR TITLE
PhoneNumberAssignment and TeamsUnassignedNumberTreatment changes

### PR DESCRIPTION
**Pull request by JensTrier**
Updating the PhoneNumberAssignment cmdlets with parameter sets, support for extensions and new example for removing location

Removing Preview label from TeamsUnassignmentNumberTreatment cmdlets

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The phone number to unnassign from the user or resource account. Supports E.164
 
 ```yaml
 Type: System.String
-Parameter Sets: (All)
+Parameter Sets: (RemoveSome)
```
Comment (get-itips) [inline teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md:69]: This parameter set does not seem to exist in the _sintax_ section

Comment (get-itips) [reply teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md:69]: It seems this is the current sintax with the correct parameter sets

image

**Comment by get-itips** [comment]
@JensTrier Jens, thanks for applying the changes, I noticed the build is giving errors now, i am not sure but maybe this extra space is the cause?

image

**Comment by JensTrier** [comment]
> @JensTrier Jens, thanks for applying the changes, I noticed the build is giving errors now, i am not sure but maybe this extra space is the cause?
> 
> image

Thanks! Yes, I also wanted to get rid of them, since I checked some other sample and they didn't have the spaces. Let's see what it says after my new edits

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**